tests/TeamleaderTest.php

```diff
@@ -2,14 +2,24 @@
 
 namespace SumoCoders\Teamleader\tests;
 
-require_once '../../../autoload.php';
-require_once 'config.php';
+// add your own credentials in this file
+require_once __DIR__ . '/credentials.php';
+
+// required to load
+require_once __DIR__ . '/../../../autoload.php';
 
 use SumoCoders\Teamleader\Teamleader;
 use SumoCoders\Teamleader\Crm\Contact;
 use SumoCoders\Teamleader\Crm\Company;
 use SumoCoders\Teamleader\Opportunities\Sale;
 
+/**
+ * Teamleader test
+ *
+ * This Teamleader PHP Wrapper class connects to the Teamleader API.
+ *
+ * @author Jeroen Desloovere <info@jeroendesloovere.be>
```

Comment:
I don't think it's really nice to add yourself here as the author, since the original authors didn't do this ;) This is kind of confusing.